manifests/00-crd-extension-console-link.yaml

```diff
@@ -52,8 +52,8 @@ spec:
               pattern: '^https://([\w-]+.)+[\w-]+(/[\w- ./?%&=])?$'
             location:
               type: string
-              description: Determines which dropdown menu the link is added to (ApplicationMenu, HelpMenu, UserMenu)
-              pattern: ^(ApplicationMenu|HelpMenu|UserMenu)$
+              description: Determines which dropdown menu the link is added to (ApplicationMenu, HelpMenu, UserMenu, NamespaceDashboard)
```

Comment:
As mentioned in planning yesterday, do we want to reconsider the location name given we intend to add these links to project and namespace kebab dropdowns, too?

Reply:
`NamespaceDashboard`... does sound vague to me.  I'm not familiar with this, but I tend to think of an entire page as a dashboard.

Reply:
We're adding it to the existing namespace dashboard page. I think it's consistent with names for other locations. Although to @rhamilto's point, if it's also in the actions, maybe we need a more generic name

Reply:
@spadgett in that case, more generic name would make sense. 
@rhamilto maybe: 
- `NamespaceLink`
- `NamespaceHelper`
- `NamespaceActionLink`
- `NamespaceDashboardAndAction`   😄
- `NamespaceCommonLink`
...
..
.

Reply:
`NamespaceLauncher`?

Reply:
@rhamilto `NamespaceLauncher` doesn't sound related to the intend this specific console link should be doing.
Most generic is `NamespaceLink` or maybe `NamespaceSharedLink`.

@spadgett thoughts ?

Reply:
My vote would be for either `location: Namspace` or `location: NamspaceDashboard`. The other locations don't have `Link` on the end, so we should be consistent in our naming.

Reply:
Then I would go with the `NamspaceDashboard` as well